controllers/cloud.redhat.com/providers/serviceaccount/utils.go

```diff
@@ -1,28 +1,29 @@
-package controllers
+package serviceaccount
 
 import (
-	"context"
-
 	crd "cloud.redhat.com/clowder/v2/apis/cloud.redhat.com/v1alpha1"
+	"cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/errors"
 	"cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/object"
+	"cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/providers"
 	"cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/utils"
 	core "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func createServiceAccount(ctx context.Context, client client.Client, obj object.ClowdObject, pullSecretNames crd.PullSecrets) error {
+func createServiceAccount(cache *providers.ObjectCache, ident providers.ResourceIdent, obj object.ClowdObject, pullSecretNames crd.PullSecrets) error {
```

Comment:
does this need to be in a separate utils file?

Reply:
no - once again consistency